lib/producer/events_producer_options.ex

```diff
@@ -62,11 +62,23 @@ defmodule Mississippi.Producer.EventsProducer.Options do
       A unique identifier for a given message. Defaults to an UUID v4.
       """
     ],
+    reconnection_backoff_ms: [
+      type: :integer,
+      doc: """
+      Backoff between reconnection attempts. Defaults to 10 seconds.
+      """
+    ],
     timestamp: [
       type: :integer,
       doc: """
       A Unix timestamp. Defaults to the current timestamp of the publish.
       """
+    ],
+    connection: [
+      type: :atom,
+      doc: """
+      AMQP Connection module.
+      """
```

Comment:
I think `reconnection_backoff_ms` and `connection` should not be publish options, as they are not related to publishing a single message on a Mississippi queue

Reply:
right, thank you!